src/UI/Implementation/Component/Input/Input.php

```diff
@@ -275,7 +298,7 @@ protected function setAdditionalConstraint(Constraint $constraint) {
 			throw new \LogicException("Can only collect if input has a name.");
 		}
 
-		$value = $input->get($this->getName());
+		$value = $input->getOr($this->getName(), null);
```

Comment:
This effectively means that we won't be able to distinguish an deliberately set null-value from a default. This will only be bad if there are (or will be) form fields that have null as a possible value. Inputs from these fields won't be processed on line 317 if the field is not required, which seems to be incorrect. Crucial question thus is: is there a type of field that may return null as a deliberately set value? Implementation will need to be different then...